diff -urN trackballs-1.0.0.org/share/Makefile.am trackballs-1.0.0/share/Makefile.am
--- trackballs-1.0.0.org/share/Makefile.am	2004-04-04 23:45:51.905208992 +0200
+++ trackballs-1.0.0/share/Makefile.am	2004-04-04 23:46:10.858327680 +0200
@@ -4,6 +4,4 @@
 EXTRA_DIST = $(pkgdata_DATA) trackballs.6
 
 install-pkgdataDATA:
-	install -D -m664 -g games highScores		$(DESTDIR)@with_highscores@
-	chgrp -R games			$(DESTDIR)$(pkgdatadir)/levels
-	chmod -R g+r			$(DESTDIR)$(pkgdatadir)/levels
+	install -D -m664 highScores		$(DESTDIR)@with_highscores@
